suites/quincy/nvmeof/ceph_nvmeof_baremetal_scale.yaml

```diff
@@ -0,0 +1,282 @@
+tests:
+# Test conf at conf/quincy/upi/octo-9-node-nvmeof.yaml
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                registry-url: registry.redhat.io
+                allow-fqdn-hostname: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  #  Test cases to be executed
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        nodes:
+          - node5
+          - node6
+          - node7
+          - node8
+          - node9
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Setup client on NVMEoF gateway
+      destroy-cluster: false
+      module: test_client.py
+      name: configure Ceph client
+      polarion-id:
+        CEPH-83573758
+
+  - test:
+    abort-on-fail: true
+    config:
+      gw_node: node5
+      rbd_pool: rbd
+      do_not_create_image: true
+      rep-pool-only: true
+      rep_pool_config:
+        pool: rbd
+      install: true
+      subsystems:
+        - nqn: nqn.2016-06.io.spdk:cnode1
+          serial: 1
+          max_ns: 64
```

Comment:
Can we have 256 namespace support for single subysystem as per TP requirenment instead of having 64.

Reply:
1 subsystem 256 namespace for subsystem - This would be my next test suite I will be adding 
4 subsystem 64 namespace for each subsystem - covered in this test suite
Above 2 are two extreme scale test scenario from TP and -  the below 2 has coverage from above 2 scenarios and this need not necessarily need separate test scenario
2 subsystem 128 namespace for each subsystem
3 subsystem 86 namespace for each subsystem

Reply:
IMO, 64 is not required for test coverage, 256 max is across the gateway. So we should ideally design as per TP and try to cover more combination from that. No need of separate test again for 256.